### ISSUE TITLE
where could i get the file "model_params_256gumbel15.tar"

### ISSUE THREAD

**Issue by sami235405** [nithinksath96/Speech_Recognition#1]
where could i get the file "model_params_256gumbel15.tar" can you please help me on file "model_params_256gumbel15.tar" or tell me how to creat the file 

path = '/home/ubuntu/hw4part2/'
path1= '/home/ubuntu/hw4part2/withoutgumbel/'
checkpoint = torch.load(path + "model_params_256gumbel15.tar")   
model.cuda()
model.load_state_dict(checkpoint['model_state_dict'])
optimizer.load_state_dict(checkpoint['optimizer_state_dict'])
print(checkpoint['loss'])
